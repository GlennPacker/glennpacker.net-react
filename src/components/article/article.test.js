import React from 'react';
import Article from './index';
import { shallow } from 'enzyme';

describe('Article', () => {
    it('renders correctly', () => {
        const wrapper = shallow(
            <Article
                heading="heading"
                content="content"
                colour="info"
                minHeight="200"
            />
        )

        expect(wrapper).toMatchSnapshot();
    })
})