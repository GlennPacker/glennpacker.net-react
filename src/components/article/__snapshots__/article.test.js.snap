// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article renders correctly 1`] = `
<div
  className="card"
  style={
    Object {
      "minHeight": "200",
    }
  }
>
  <h5
    className="bg-info card-header text-white"
  >
    heading
  </h5>
  <div
    className="card-body"
  >
    content
  </div>
</div>
`;
